RecoLocalCalo/HcalRecAlgos/src/HFSimpleTimeCheck.cc

```diff
@@ -0,0 +1,133 @@
+#include <cstring>
+#include <climits>
+
+#include "RecoLocalCalo/HcalRecAlgos/interface/HFSimpleTimeCheck.h"
+#include "RecoLocalCalo/HcalRecAlgos/interface/HFRecHitAuxSetter.h"
+
+HFSimpleTimeCheck::HFSimpleTimeCheck(const std::pair<float,float> tlimits[2],
+                                     const float energyWeights[2*HFAnodeStatus::N_POSSIBLE_STATES-1][2],
```

Comment:
> On Jun 17, 2016, at 11:22 AM, Igor Volobouev <notifications@github.com> wrote:
> 
> In RecoLocalCalo/HcalRecAlgos/src/HFSimpleTimeCheck.cc:
> 
> > @@ -0,0 +1,133 @@
> > +#include <cstring>
> > +#include <climits>
> > +
> > +#include "RecoLocalCalo/HcalRecAlgos/interface/HFSimpleTimeCheck.h"
> > +#include "RecoLocalCalo/HcalRecAlgos/interface/HFRecHitAuxSetter.h"
> > +
> > +HFSimpleTimeCheck::HFSimpleTimeCheck(const std::pair<float,float> tlimits[2],
> > +                                     const float energyWeights[2*HFAnodeStatus::N_POSSIBLE_STATES-1][2],
> 
> Yes, but passing by reference brings no advantage for the simple use of arrays in this code, and any code change now would mean another week-long delay for collecting signatures

it would cause little delay as we don’t wait for signatures a second time around for simple changes.

how small is this array?


> …
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub, or mute the thread.
>